### DISCUSSION TITLE
입장 질문하나만더 ㅠㅠ

### DISCUSSION THREAD

**Discussion by leechangseok917** [storycraft/node-kakao#373] [Q&A]
const openChannelList = CLIENT.channelList.open;

**
openChannelList.joinChannel(172574203,1,"");

**

안되는데
뭐가틀린걸까요

**Comment by raravel** [comment]
joinChannel 의 인자 전달 1, 2번이 틀렸습니다.

#344 예제 참고하세요.

**Reply by leechangseok917** [reply]
참고해도 모르겠는데...

링크id를 저렇게 입력하는게 아닌가요?

**Reply by leechangseok917** [reply]
v3당시에는 저런식으로 접속했었던걸로 기억합니다... ㅠㅠ

**Reply by raravel** [reply]
참고해도 모르겠는 건 개인에 따라 다르기때문에 어떻게 해드릴 수 있는 건 없습니다.

v3를 사용하실 거면 npm에서 3.x.x 번대를 다운받아 사용하시면 됩니다.

https://www.npmjs.com/package/node-kakao/v/3.1.10

**Reply by leechangseok917** [reply]
감사합니다!!

[a comment by an automated account is not shown]
